notebooks/demo2/transformer-inference-sparse.ipynb

```diff
@@ -0,0 +1,1141 @@
+{
```

Comment:
<p>These times look like too good to be true! Why are there multiple rows for the same model name and same pdf?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/os-climate/aicoe-osc-demo/pull/234/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>